assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -362,6 +366,19 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
       .attr('xlink:target', '_blank')
       .attr('xlink:href', link)
     a.node().appendChild(this)
+    
+
+    let resourceType = d.split('|')[2]
+    d3.select(this).insert('foreignObject')
+      .attr("x", -90)
+      .attr('y', -6)
+      .attr("width", 32)
+      .attr("height", 32)
+      .append("xhtml:i")
+      .attr('class', function(_) {
+        return icons[resourceType]
```

Comment:
I think if you pulled these icons out of the config, they would be in placed in the top level myla_globals.